pricing-page-exercise/saffet/index.html

```diff
@@ -1,13 +1,138 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="./styles.css">
-  <title>Pricing page Exercise</title>
+  <title>Pricing example for Bootstrap (without Bootstrap)</title>
+  <link rel="icon" href="icon_path" type="image/icon type">
 </head>
+
 <body>
-  
+
+  <div class="nav-bar">
+
+    <p>Company name</p>
```

Comment:
I'd say it's more of a span element when you think about semantics. In reality, it would always be a link element on a page like this.

Reply:
"Company name" replaced with a <a> tag instead of <p>